yarn.lock

```diff
@@ -1401,8 +1401,7 @@
     uuid "^3.3.2"
 
 "@elastic/datemath@link:packages/elastic-datemath":
-  version "0.0.0"
-  uid ""
```

Comment:
@spalger is this expected? Seems a bit strange.

Reply:
Yeah, it seems that renovate isn't doing the right thing here, when I make a change to the yarn.lock file locally all those changes get reverted :/ I'll fix up this PR and file an issue with them.